src/rest_api/classes/blast_hit.clj

```diff
@@ -0,0 +1,66 @@
+(ns rest-api.classes.blast-hit
+  (:require
+   [compojure.api.sweet :as sweet]
+   [datomic.api :as d]
+   [rest-api.db.main :refer [datomic-conn]]
+   [rest-api.formatters.object :as obj :refer [pack-obj]]
+   [ring.util.http-response :as response]
+   [schema.core :as schema]))
+
+(def id-types [:transcript/id :cds/id :sequence/id :protein/id])
+
+(defmulti convert-entity
+  (fn [entity]
+    (first (filter (partial contains? entity) id-types))))
+
+;; wormpep
+(defmethod convert-entity :transcript/id [entity]
+  (let [transcript entity
```

Comment:
redundant alias; rename the `entity` parameter to `transcript` or just use `entity`